src/services/storage/models/agreement.model.ts

```diff
@@ -20,6 +20,7 @@ export default class Agreement extends Model {
   @Column({ type: DataType.STRING(64) })
   consumer!: string
 
+  // In Megabytes
```

Comment:
Interesting, I guess since we don't do any calculations with external data we don't have to change it anywhere else right? 🤔

Reply:
You are right. Just add comments for future